content/3_Silver/Two_Pointers.mdx

```diff
@@ -280,3 +280,93 @@ public class Books {
 ## Problems
 
 <Problems problems="ad" />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    What is the two pointers algorithm?
+    <Quiz.Answer correct>
+      Keeping track of two pointers to find a pair of indices that satisfy some condition.
+      <Quiz.Explanation>
+        Correct.
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A way to answer range sum queries in $\mathcal{O}(1)$ time.
+      <Quiz.Explanation>
+        Incorrect. You might be thinking of [Prefix Sums](/silver/prefix-sums), another silver topic.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      A divide and conquer algorithm that splits an array into two halves to efficiently find an element satisfying a certain condition.
+      <Quiz.Explanation>
+        Incorrect.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    When would a solution be impossible when finding two distinct elements in an array which add up to a target while doing a collision two pointers algorithm? (two pointers move from the ends of the array to the middle). 
+
+	```
+	// find two distinct elements in an array 
+	
+	nums.sort()
+
+	l = 0
+	r = n - 1
+
+	while (???) {
+		if (nums[i] + nums[r] == target) {
+			found pair
+			break
+		}
+		else if (nums[l] + nums[r] > target) {
+			r -= 1
+		} else {
+			l += 1
+		}
+	}
+	```
+    <Quiz.Answer>
+      If the array isn't sorted.
```

Comment:
this answer choice no longer makes sense. maybe just ask what should go in place of `???`?

Reply:
unresolved, now none of the answer choices make sense :/

Reply:
hopefully this makes more sense, sorry for the late response